src/app/static/src/app/components/header/DataExplorationHeader.vue

```diff
@@ -2,24 +2,23 @@
     <header class="mb-5">
         <nav class="navbar navbar-dark bg-secondary">
             <div class="container-fluid">
-                <div class="navbar-header">
+                <div class="navbar-header-secondary">
                     {{ title }}
                 </div>
-                <div style="flex: 1 1 auto;" class="ml-2"><a href="/">Run model</a></div>
+                <div style="flex: 1 1 auto;" class="ml-2"><a href="/" v-translate="'runModel'"></a></div>
                 <span v-if="!isGuest" class="pr-2 mr-2 border-right text-white">
                     <span v-translate="'loggedInAs'"></span> {{ user }}
                 </span>
                 <hintr-version-menu class="pr-2 mr-2 border-right"/>
-                <online-support-menu class="pr-2 mr-2 border-right"/>
                 <a :href="'public/resources/' + helpFilename"
                    id="helpFile"
                    target="_blank"
                    class="pr-2 mr-2 border-right"
                    v-translate="'dataExplorationHelp'">
                 </a>
-                <a v-if="!isGuest" href="/logout" class="pr-2 mr-2 border-right" v-translate="'logout'">
+                <a v-if="!isGuest" :href="'/logout'" class="pr-2 mr-2 border-right" v-translate="'logout'">
```

Comment:
Shouldn't need the colon notation for these hrefs, they don't need to be evaluated since they won't change

Reply:
colon is not needed to get it working but required to resolve path

Reply:
Sorry, I don't know what you mean by that. I just tried making that change and it works fine manually, and the tests still pass, so not sure what the problem is?

Reply:
I meant it does work fine without colon.

The reason for adding colon is because intelliJ complains about the file reference as it is not getting resolved. Changing href to a computed href means it does not see file directory as plain html but computed value, hence issue resolved.

Reply:
Well, ok - but we do have other things in the templates that IntelliJ low-key complains about e.g 'typos' in class names, and it's easy enough to suppress that warning, which isn't really relevant for us.